resolwe_bio/processes/junctions/regtools.yml

```diff
@@ -0,0 +1,119 @@
+# =========================================
+# Regtools - Junctions Extract and Annotate
+# =========================================
+---
+
+- slug: junctions-annotate
+  name: Annotate novel splice junctions (regtools)
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/rnaseq:regtools
+  data_name: "Novel Junctions ({{ some input }})"
+  version: 0.0.1
+  type: data:junctions:novel
```

Comment:
Since the process creates the .bed file, make the type `data:bed:junctions:novel`